### ISSUE TITLE
Pretty sure I got permently banned

### ISSUE THREAD

**Issue by heffman92** [finndev/PokeBuddy#726]
When I open the app there's nothing. Can't see Pokestops, sightings, gyms. Nothing

**Comment by ukyo28** [comment]
please read other issues before posting a new one.

You are on IP ban

**Comment by ukyo28** [comment]
#707

**Comment by heffman92** [comment]
OK. I wasn't sure if it did that on the App as well

**Comment by barbaricacidman** [comment]
yep, if you turn your wifi off and use your mobile data everything will come back if you reload the game

**Comment by 404unknown** [comment]
Only run 1 bot on 1 device. Agree with @ukyo28 , Read other issues before opening another duplicate post

**Comment by Thorben24** [comment]
If there are no pokestops on the map it is an IP-Ban. You have to change your IP adress in this case to solve this problem. 95% of the providers use dynamic ip adressing, so a relog / restart of your router should renew your ip adress. Usually the IP ban will only happen if you bot / play more than 1 account at the same time. In the future you should only use 1 ip adress for 1 account. For others you would have to use proxy tunnels each pokebuddy instance you are running. Right now, if you can not switch your ip adress, you can login via a VPN. Here is a great tutorial for that: You can also just use a vpn for logging in and switching back to your home adress after it. Here is a great tutorial: http://www.ownedcore.com/forums/pokemon-go/pokemon-go-hacks-cheats/570331-quick-ip-unban-vpn-ip-no-ban.html

**Comment by TankardBR** [comment]
I tested tonight 3 accounts for about 7 hours , when I woke up in the morning to look had not been banned any account but the bot was not finding any pokemon or pokestop . Must have working for 2 hours as it could only 140k XP. I Start again and we'll see what happens. Sorry about my English.

**Comment by Thorben24** [comment]
Not finding Pokestops means your account is ip banned. If this is some hours ago, it should work now.

**Comment by TankardBR** [comment]
I did some tests 3 accounts  5-30 min ip ban, 2 accounts 5-30 min ip ban, 1 account works normally. Changing the GPS from one country to another instant softban. Also I figured out how to get out of softban. And the bot slow to farm, I am only farming 20-25k per hour before i picked up 70k. Sorry about my English.
